bin/linguist

```diff
@@ -16,14 +17,17 @@ end
 
 ARGV.shift
 breakdown = true if ARGV[0] == "--breakdown"
+json_breakdown = true if ARGV[0] == "--json"
```

Comment:
Should it be `--breakdown --json` for a full JSON breakdown and `--json` just to get statistics as JSON?